### ISSUE TITLE
Can names contain spaces?

### ISSUE THREAD

**Issue by KevinWorkman** [Priya411/CodeUGroup3#22]
https://github.com/Priya411/CodeUGroup3/blob/master/src/codeu/chat/client/commandline/Chat.java#L449

I haven't actually tested this yet, but since we're joining on space, wouldn't spaces in usernames be lost?

If I want my username to be `Kevin Workman`, would this logic force my username to be `KevinWorkman` instead?

Same thing for conversation names.

Like I said I haven't actually tested this so maybe I'm wrong. And even if I'm right, maybe disallowing spaces is a feature. :p

**Comment by TheMkrage** [comment]
No, the tokenizer separates by spaces and commas, we just append the args together with this command after they are tokenized into separate args. so the only thing lost is something like "Matthew,Krager" becomes "Matthew Krager" but spaces are allowed in both.
